### PR TITLE
Properly version settings-tp package

### PR DESCRIPTION
**Pull request by rbuysse**

### CODE REVIEW

File: families/settings/Dockerfile-installed-xenial [hunk 1]
```diff
@@ -25,9 +25,11 @@ RUN apt-get update \
  && apt-get install -y \
  curl \
  gcc \
+ git \
```
Comment (peterschwarz) [inline families/settings/Dockerfile-installed-xenial]: Why is Git needed? Is it used in a script not part of this PR?

Comment (rbuysse) [reply families/settings/Dockerfile-installed-xenial]: The bin/get_version script runs `git describe` to determine the real version of the repository.
